iohk/NixOps.hs

```diff
@@ -73,6 +75,9 @@ projectURL     IOHK            = "https://github.com/input-output-hk/iohk-nixops
 projectURL     Nixpkgs         = "https://github.com/nixos/nixpkgs.git"
 projectURL     Stack2nix       = "https://github.com/input-output-hk/stack2nix.git"
 
+nixpkgsRevTarball :: Commit -> URL
+nixpkgsRevTarball (Commit rev) = URL $ "https://github.com/NixOS/nixpkgs/archive/"<>rev<>".tar.gz"
```

Comment:
That would make us enter all of those values for all repositories served by `set-rev`, and to me that looks highly unergonomic.

To resolve that we'd need defaulting.  To do defaulting we'd need to extend the URL type.